frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelinesTable.ts

```diff
@@ -11,7 +11,11 @@ class PipelinesTable {
   }
 
   findRowByName(name: string) {
-    return this.find().findAllByRole('heading', { name });
+    return this.find().findAllByRole('heading', { name }).closest('tr');
+  }
+
+  getRowByName(name: string) {
```

Comment:
I think this can be removed and all usages be replaced with `findRowByName`.

Reply:
I need to verify something does not exist, and `findRowByName` will throw the error because it couldn't find the element.

Reply:
I see. The problem I have with it then is that someone could easily be confused as to which should be used to get the row, since find/get are synonymous. Maybe this "getRow" could be more of a "hasRow", and the usage could be more of a boolean expectation.